packages/appstore/src/components/add-options-account/add-options-account.tsx

```diff
@@ -29,6 +30,12 @@ const AddOptions = observer(() => {
                     type='submit'
                     has_effect
                     onClick={() => {
+                        Analytics.trackEvent('ce_tradershub_dashboard_form', {
+                            action: 'account_get',
+                            form_name: 'traders_hub_default',
+                            account_mode: document.getElementById('dropdown-display')?.innerText,
```

Comment:
@aswathy-deriv can we use any state to get the account mode here instead of `document.getElementById` ?

Reply:
In the appstore package 'Demo' and 'Real' are categorized as `selected_account_type` which can be gotten from `traders_hub` store. Maybe better to change it to below

```suggestion
    const { selected_account_type } = traders_hub;

                            selected_account_type: selected_account_type,
```

Reply:
Thanks @mahdiyeh-deriv